DhawaAlkhreej/DhawaAlkhreej/Modules/OnBoarding/PageController.swift

```diff
@@ -32,30 +32,98 @@ class PageController: BaseViewController {
                 self.getStartedButton.alpha = 1
             })
         }
+        Helper.feedbackGenerator(type: .light)
+        animateImageView()
     }
-
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         
     }
     override func setupUI() {
         setupGetStartedButton()
+        setupImageView()
         textLabel.text = text
-        imageView.image = UIImage(named: imageName)
-        getStartedButton.alpha = 0
     }
     
     private func setupGetStartedButton() {
-        
+        getStartedButton.alpha = 0
         getStartedButton.layer.cornerRadius = 10
         getStartedButton.layer.shadowOffset = CGSize.zero
         getStartedButton.layer.shadowOpacity = 0.2
         getStartedButton.titleLabel?.font = Identity.font(.custom(weight: .bold, size: 20))
+        
+        addMotionEffects(to: getStartedButton, shadow: true, movement: true)
+    }
+    
+    private func setupImageView() {
+        
+        imageView.image = UIImage(named: imageName)
+        imageView.layer.shadowOffset = CGSize.zero
+        imageView.layer.shadowOpacity = 0.2
+        addMotionEffects(to: imageView, shadow: true, movement: true)
     }
     
     @IBAction func getStartedButtonPressed(_ sender: Any) {
         UserDefaults.standard.set(true, forKey: K.UserDefaults.onBoardingViewed)
+        Helper.feedbackGenerator(type: .medium)
         dismiss(animated: true, completion: nil)
     }
+}
+
+// Some Helper Methods
+extension PageController {
+    
+    private func animateImageView() {
```

Comment:
All of this methods could be as an extension to the `UIIView` and it can be used in other places